src/Microsoft.ML.Transforms/Text/StopWordsRemoverTransform.cs

```diff
@@ -286,121 +187,44 @@ private static NormStr.Pool[] StopWords
             }
         }
 
-        public StopWordsRemoverTransform(IHostEnvironment env, Arguments args, IDataView input)
-            : base(env, RegistrationName, Contracts.CheckRef(args, nameof(args)).Column, input, TestIsTextVector)
+        public class ColumnInfo
```

Comment:
> publi [](start = 7, length = 6)

doc

Reply:
also make it `sealed`

---
In reply to: [227843842](https://github.com/dotnet/machinelearning/pull/1338#discussion_r227843842) [](ancestors = 227843842)